src/components/home/approach.tsx

```diff
@@ -189,25 +222,55 @@ const StyledIcon = styled(ArrowUpRight)`
 
   transition: 0.3s ease;
 
-  &:hover {
-    transform: rotate(45deg);
-  }
 
   @media all and (max-width: ${breakpoints.md}px) {
-    top: 38%; /* adjust top for medium screens */
-    right: 6%; /* adjust right for medium screens */
+    top: 38%;
+    right: 6%;
   }
 
   @media all and (max-width: ${breakpoints.sm}px) {
-    top: 29%; /* adjust top for small screens */
-    right: 6%; /* adjust right for small screens */
+    top: 30%;
+    right: 4%;
+`;
+
+const StyledPillButton = styled(motion.div)`
+  width: 47%;
+  max-width: 777px;
+  min-width: 237px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 24px;
+  background-color: ${colors.white};
+  color: ${colors.black};
+  font-weight: 500;
+  border-radius: 26px;
+  padding: 4vw;
+  font-size: 25px;
+  letter-spacing: 2px;
+  display: flex;
+  align-items: center;
+  position: relative;
+  border: 2px solid rgba(29, 29, 29, 0.1);
+
+  &:hover .styled-icon {
```

Comment:
```suggestion
cursor: pointer;
& hover {
background-color: ${colors.accent};
}
  &:hover .styled-icon {
```

Reply:
so just when you hover your mouse the button should do as normal buttons do e.g. change colour and the cursor should be a pointer